README.adoc

```diff
@@ -19,12 +19,15 @@
 :page-related-guides: ['microprofile-health', 'kubernetes-intro']
 :common-includes: https://raw.githubusercontent.com/OpenLiberty/guides-common/master
 :source-highlighter: prettify
-= Probing microservices running in Kubernetes
+:page-seo-title: Kubernetes readiness and liveness probe tutorial
+:page-seo-description: How to setup liveness probes and readiness probes in Kubernetes
```

Comment:
The SEO title and description should map to what we have in the guide too.  We can build upon what we have for the actual title and abstract.

Reply:
What we have done for recent guides, which is similar to what you have, isn't ideal, @proubatsis.